docs/examples/DataSet/Accessing-data-in-DataSet.ipynb

```diff
@@ -0,0 +1,1051 @@
+      "text/plain": [
+       "'x,t,y,y2,q'"
+      ]
+     },
+     "execution_count": 24,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dataset.parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To give an example of what it takes to work with `ParamSpec` objects as opposed to `Interdependencies_` object, here's a function that one needs to write in order to find standalone `ParamSpec`s from a given list of `ParamSpec`s:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(ParamSpec('q', 'numeric', 'Qredibility', '$', inferred_from=[], depends_on=[]),)"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def get_standalone_parameters(paramspecs):\n",
+    "    all_independents = set(spec.name\n",
+    "                           for spec in paramspecs\n",
+    "                           if len(spec.depends_on_) == 0)\n",
+    "    used_independents = set(d for spec in paramspecs for d in spec.depends_on_)\n",
+    "    standalones = all_independents.difference(used_independents)\n",
+    "    return tuple(ps for ps in paramspecs if ps.name in standalones)\n",
+    "\n",
+    "all_parameters = dataset.get_parameters()\n",
+    "standalone_parameters = get_standalone_parameters(all_parameters)\n",
+    "standalone_parameters"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting data from DataSet\n",
+    "\n",
+    "In this section methods for retrieving the actual data from the `DataSet` are discussed.\n",
+    "\n",
+    "### `get_parameter_data` - the powerhorse\n",
+    "\n",
+    "`DataSet` provides one main method of accessing data - `get_parameter_data`. It returns data for groups of dependent-parameter-and-its-independent-parameters in a form of a nested dictionary of `numpy` arrays:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'q': {'q': array([3.        , 2.08558738, 2.259722  , 3.31510822, 3.99537911,\n",
+       "         3.49071755, 2.40188947, 2.02507209, 2.80884137, 3.82017225,\n",
+       "         3.85514276, 2.87212284, 2.04133215, 2.3517716 , 3.43388374,\n",
+       "         3.99948622, 3.375267  , 2.30431745, 2.06153158, 2.93592978,\n",
+       "         3.88659931, 3.78183148, 2.74634542, 2.01281822, 2.4544651 ,\n",
+       "         3.5455349 , 3.98718178, 3.25365458, 2.21816852, 2.11340069,\n",
+       "         3.06407022, 3.93846842, 3.69568255, 2.624733  , 2.00051378,\n",
+       "         2.56611626, 3.6482284 , 3.95866785, 3.12787716, 2.14485724,\n",
+       "         2.17982775, 3.19115863, 3.97492791, 3.59811053, 2.50928245,\n",
+       "         2.00462089, 
```

Comment:
Again the ticks break the rendered link. I also suggest linking to the latest version of the docs with https://pandas.pydata.org/pandas-docs/stable/reference/frame.html#serialization-io-conversion 

At the moment the latest release is 0.25.2 so the above link is already out of date and the stable link seems like a better bet.

Reply:
thanks for the link! didn't find it myself ((